cpp/oneapi/dal/table/csr.cpp

```diff
@@ -37,6 +40,27 @@ sparse_indexing csr_table::get_indexing() const {
 
 csr_table::csr_table() : csr_table(new backend::csr_table_impl{}) {}
 
+csr_table::csr_table(const table& other) {
```

Comment:
There is more correct implementation in PR #2571. Please wait for it to be merged.

Reply:
Yes, It's a blocker for current PR